dateparser/date.py

```diff
@@ -228,6 +228,16 @@ def _is_valid_date_obj(self, date_obj):
 class DateDataParser(object):
 
     def __init__(self, languages=None, allow_redetect_language=False):
+        """
+        :param languages:
+                A list of two letters language codes.e.g. ['en', 'es'].
+                If languages are given, it will not attempt to detect the language.
+
+        :param allow_redetect_language:
+                Enables/disables language re-detection.
+        
+        :return: A parser instance
+        """
```

Comment:
Hey fellows! :)

The docstring appear in the docs, they're just not referenced much in the main docs text -- they're collected by Sphinx `autodocs` extension, enabled in `docs/conf.py`.

If you go to the module index: http://dateparser.readthedocs.org/en/latest/py-modindex.html
you can click in a module and it'll take you to that module's docs: http://dateparser.readthedocs.org/en/latest/dateparser.html